src/modules/airdrop/services/airdrop-service.ts

```diff
@@ -26,10 +26,10 @@ export class AirdropService {
     private userService: UserService,
   ) {}
 
-  public async getRewards(walletAddress: string) {
+  public async getRewards(walletAddress: string, userId?: number) {
     const checksumAddress = ethers.utils.getAddress(walletAddress);
     const walletRewards = await this.getWalletRewards(checksumAddress);
-    const communityRewards: string = await this.getCommunityRewards(checksumAddress);
+    const communityRewards: string = await this.getCommunityRewards(userId);
```

Comment:
NIT: Do we have to cast this to `string` even though it can be `undefined`? I mean are handling the `undefined` case correctly below AFAICT.

Reply:
Removed the cast. We don't need it